privacyidea/api/auth.py

```diff
@@ -365,11 +365,11 @@ def get_auth_token():
                         "authtype": authtype,
                         "exp": datetime.utcnow() + validity,
                         "rights": rights},
-                       secret, algorithm='HS256').decode('utf8')
+                       secret, algorithm='HS256')
```

Comment:
Shouldn't we move this hidden fix to an extra issue/PR?
Or is this due to a change in the requirements.

Reply:
The `jwt.encode()` returns a unicode string for Python > 3.5 so the `.decode()` fails. I have added the `to_unicode(token)` later to always return a unicode string.
This is due to an update in PyJWT.